commands/firstline/index.js

```diff
@@ -29,27 +29,38 @@ module.exports = {
 			};
 		}
 
+		const targetChannel = (channel)
+			? sb.Channel.get(channel)
+			: context.channel;
+
+		if (!targetChannel) {
+			return {
+				success: false,
+				reply: "Provided channel not found in the database!"
+			}
+		}
+
 		let metaData = await sb.Query.getRecordset(rs => rs
 			.select("First_Message_Posted", "First_Message_Text")
 			.from("chat_data", "Message_Meta_User_Alias")
 			.where("User_Alias = %n", targetUser.ID)
-			.where("Channel = %n", context.channel.ID)
+			.where("Channel = %n", targetChannel.ID)
 			.limit(1)
 			.single()
 		);
 
 		if (!metaData) {
 			return {
 				success: false,
-				reply: "That user has not said anything in this channel!"
+				reply: "That user has not said anything in the specified channel!"
```

Comment:
Instead of mentioning `specified` every time, create a `const` variable that is either `"specified"` or `"this"` based on if the `targetChannel` is equal to `context.channel`

Reply:
I have renamed `"the specified"` to `"that"` now.

So, if the `targetChannel` is equal to `context.channel`
-> "this channel"

Else
->  "that channel"